core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -493,6 +493,17 @@ public String getUrl() {
         return url;
     }
 
+    /**
+     *
+     * @return the URL used by {@link jenkins.install.SetupWizard} for suggested plugins to install at setup time
+     * @since 2.445 (ish)
```

Comment:
because I'm not sure when this will be merged.. :)

Reply:
```suggestion
     * @since TODO
```